packages/core/package.json

```diff
@@ -2,59 +2,47 @@
   "name": "monday-ui-react-core",
   "version": "2.116.0",
   "description": "Official monday.com UI resources for application development in React.js",
-  "main": "./dist/main.js",
-  "types": "./dist/types.d.ts",
+  "types": "./dist/index.d.ts",
   "exports": {
-    "./package.json": "./package.json",
     ".": {
-      "types": "./dist/types.d.ts",
-      "import": "./dist/esm/src/index.js",
-      "default": "./dist/main.js"
+      "types": "./dist/index.d.ts",
+      "import": "./dist/src/index.js"
     },
-    "./dist/": "./dist/",
     "./icons": {
-      "types": "./dist/esm/components/Icon/Icons/index.d.ts",
-      "import": "./dist/esm/src/components/Icon/Icons/icons.js",
-      "default": "./dist/allIcons.js"
+      "types": "./dist/components/Icon/Icons/index.d.ts",
+      "import": "./dist/src/components/Icon/Icons/icons.js"
     },
     "./interactionsTests": {
-      "types": "./dist/esm/tests/interactions-utils.d.ts",
-      "import": "./dist/esm/src/tests/interactionsTests.js",
-      "default": "./dist/esm/src/tests/interactionsTests.js"
+      "types": "./dist/tests/interactions-utils.d.ts",
+      "import": "./dist/src/tests/interactionsTests.js"
     },
     "./testIds": {
-      "types": "./dist/esm/tests/test-ids-utils.d.ts",
-      "import": "./dist/esm/src/tests/testIds.js",
-      "default": "./dist/esm/src/tests/testIds.js"
+      "types": "./dist/tests/test-ids-utils.d.ts",
+      "import": "./dist/src/tests/testIds.js"
     },
-    "./storybookComponents": "./dist/storybook/index.js",
     "./next": {
-      "types": "./dist/esm/components/next/next.d.ts",
-      "import": "./dist/esm/src/components/next/next.js",
-      "default": "./dist/esm/src/components/next/next.js"
+      "types": "./dist/components/next/next.d.ts",
+      "import": "./dist/src/components/next/next.js"
     },
     "./tokens": "./dist/tokens/tokens.css",
+    "./colors": "./dist/assets/colors.json",
     "./mockedClassNames": {
-      "import": "./dist/mocked_classnames_esm/src/index.js"
-    },
-    "./types": "./dist/esm/types/index.d.ts"
```

Comment:
Not needed anymore, those types are now exported from the root entry and we also need to do some cleanup there